### ISSUE TITLE
The test accuracy is pretty low (53.5%) for citeseer dataset

### ISSUE THREAD

**Issue by erichanxin** [Diego999/pyGAT#58]
I switched the dataset to citeseer, and trained for 100 epochs, the 99th epoch training log is:
Epoch: 0099 loss_train: 0.9922 acc_train: 0.7000 loss_val: 1.0769 acc_val: 0.7100 time: 38.2587s

However, the finial test result is:
Test set results: loss= 1.2956 accuracy= 0.5350

The accuracy is pretty low, not like the 72% shown in the paper

**Comment by SuperBruceJia** [comment]
> I switched the dataset to citeseer, and trained for 100 epochs, the 99th epoch training log is:
> Epoch: 0099 loss_train: 0.9922 acc_train: 0.7000 loss_val: 1.0769 acc_val: 0.7100 time: 38.2587s
> 
> However, the finial test result is:
> Test set results: loss= 1.2956 accuracy= 0.5350
> 
> The accuracy is pretty low, not like the 72% shown in the paper

Have you solved the problem?

**Comment by MichailChatzianastasis** [comment]
Is there any update on this issue? I also obtain low performance on citeseer.

**Comment by erichanxin** [comment]
Actually NO! I cannot find any method to solve this problem.

**Comment by 1ofsomepeople** [comment]
NVIDIA GeForce RTX 3090
CUDA version: 11.4
Pytorch version: 1.10.1+cu111
Default arguments

**Experiment Result On Citeseer**
Loading 393 epoch
Test set results: loss= 1.1030 accuracy= 0.6120

**Comment by LQchen1** [comment]
Based on your epoch log, it looks like the model is overfitting, maybe try changing the dropout parameter
